<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordPage</class>
 <widget class="QDialog" name="ChangePasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ChangePasswordPage{
background-color: #eaf2f8;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>رمز فعلی</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentPasswordField">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#currentPasswordField {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #currentPasswordField:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #returnButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #returnButton:hover {
        background-color: #3498db;
    }
    #returnButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3 {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>تایید رمز جدید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmNewPasswordField">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#confirmNewPasswordField {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #confirmNewPasswordField:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordType">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#passwordType {
        color: #2c3e50;
        font-size: 20px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordField">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#newPasswordField {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #newPasswordField:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_4 {
        color: #2c3e50;
        font-size: 20px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>تغییر رمز</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #okButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #okButton:hover {
        background-color: #3498db;
    }
    #okButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>تایید</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2 {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>رمز جدید</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
